docs/dev-guide/contributing.md

```diff
@@ -208,8 +214,17 @@ npx webpack-bundle-analyzer build/app-stats.json
 
 ### Kotlin
 
-For Kotlin code we use the [standard convention](https://kotlinlang.org/docs/coding-conventions.html) and limit line length to 120 characters. We use `ktlint` to enforce formatting.
+- For Kotlin code we use the [standard convention](https://kotlinlang.org/docs/coding-conventions.html) and limit line length to 120 characters. We use `ktlint` to enforce formatting.
+
+## ✅ Code Reviews
+
+- **Submit Your Contribution:** After completing your work, submit your contribution.
```

Comment:
Let's add some text to also encourage Draft PRs if an approach should be discussed. It's frustrating to finish some work only to scrap it later. Better to open a draft PR early for discussion.